### PR TITLE
<br/> is double rendered - renderNode vs renderSpecialNode

### PR DESCRIPTION
**Pull request by mini-eggs**
I'm not as confident in this solution as I was with the last PR (pull 43). I've added another conditional within a case in the switch statement in the traverse function (render/traverse.js) along with adding a function (renderSpecialNode). Also, the array specialNodes in the same file may need to be added to in the future.

Lemme know, yo. 👍

### CODE REVIEW

**Comment by mini-eggs** [comment]
Check issue info on https://github.com/FormidableLabs/rapscallion/issues/44

**Comment by aweary** [comment]
There are a number of elements in which the closing tag should be omitted. React [tracks these internally](https://github.com/facebook/react/blob/0585ee8b22069f20797d3c4d210d7e2d1f62a416/src/renderers/dom/stack/client/ReactDOMComponent.js#L353-L371) and it's probably best to take a similar approach.

I would prefer a condition in `renderNode` over a new ~mostly identical function.

**Comment by mini-eggs** [comment]
Great link. Thank you.

I opted for a new func due to three lines in renderNode needing to be changed (the first two lines and the last line). I can understand why a conditional in the renderNode func would be good, though.

I'll go ahead and put some conditional in the renderNode func along with adding to the specialNodes array.

**Comment by mini-eggs** [comment]
Hope this is up to par. I'm new to the whole open source thing. Let me know if the code quality isn't there/what I can do to improve it. Thank you 8-)

File: src/render/traverse.js [hunk 1]
```diff
@@ -40,13 +57,15 @@ function renderNode (seq, node, context) {
   seq.emit(() => `<${node.type}`);
   seq.emit(() => renderAttrs(node.props, seq));
   seq.emit(() => REACT_ID);
-  seq.emit(() => ">");
+  seq.emit(() => omittedCloseTags.includes(node.type) ? "/>" : ">");
```
Comment (wmertens) [inline src/render/traverse.js]: Isn't it faster to convert this array to `const isOmitted = {area: true, base: true, ...}` and then use `isOmitted[node.type]`?

Comment (mini-eggs) [inline src/render/traverse.js]: Yes, I'll push another commit fixing this up. Thanks!

**Comment by mini-eggs** [comment]
Sounds swell to me! Will do @divmain

**Pull request closed without merging**